src/libraries/System.Private.CoreLib/src/System/IO/Enumeration/FileSystemEnumerator.Windows.cs

```diff
@@ -75,6 +77,84 @@ private void Init()
             }
         }
 
+        /// <summary>
+        /// Fills the buffer with the next set of data.
+        /// </summary>
+        /// <returns>'true' if new data was found</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private unsafe bool GetData()
```

Comment:
I assume all this code is exactly as it was in the other file?

Reply:
Yes